src/components/PlusButtonEdge.tsx

```diff
@@ -1,7 +1,7 @@
 import './PlusButtonEdge.css';
 import { MiniCatalog } from '@kaoto/components';
-import { findStepIdxWithUUID, insertableStepTypes, insertStep } from '@kaoto/services';
-import { useIntegrationJsonStore, useNestedStepsStore } from '@kaoto/store';
+import { StepsService, ValidationService } from '@kaoto/services';
+import {useIntegrationJsonStore, useNestedStepsStore, useVisualizationStore} from '@kaoto/store';
```

Comment:
Make sure you have Prettier formatting set up in your IDE (see below for example on WebStorm).. though I might just add this to part of the CI workflow, or to automatically format the code on push.

![Screen Shot 2023-02-10 at 3 14 04 pm](https://user-images.githubusercontent.com/3844502/218130414-e2c08391-a9f3-4f57-ba08-2adb89ec72a7.png)